pantheon-bundle/src/main/java/com/redhat/pantheon/auth/sso/LoginServlet.java

```diff
@@ -0,0 +1,52 @@
+package com.redhat.pantheon.auth.sso;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.servlets.SlingAllMethodsServlet;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.Servlet;
+import java.io.IOException;
+
+/**
+ * A login servlet that handles Keycloak or basic auth request
+ *
+ * @author Lisa Davidson
+ */
+@Component(
+        service = Servlet.class,
+        property = {
+                "sling.servlet.methods={GET, POST}",
```

Comment:
minor: there is a `@SlingServlet` (or something like that) annotation that makes this a bit cleaner.

Reply:
updated!

Reply:
according to this documentation, osgi annotation is preferred.
  The @SlingServlet annotation (evaluated by maven-scr-plugin, use only if you can neither use 1. nor 2.)
https://sling.apache.org/documentation/the-sling-engine/servlets.html